src/ray/raylet/worker_pool.h

```diff
@@ -528,6 +550,15 @@ class WorkerPool : public WorkerPoolInterface, public IOWorkerPoolInterface {
   /// worker types (SPILL_WORKER and RESTORE_WORKER and UTIL_WORKER).
   bool IsIOWorkerType(const rpc::WorkerType &worker_type);
 
+  inline void PopWorkerCallbackAsync(const PopWorkerCallback &callback,
+                                     std::shared_ptr<WorkerInterface> worker,
+                                     Status status = Status::OK());
+
+  void TryToCallbackTask(
```

Comment:
```suggestion
  void TryToAssignTaskToDedicatedWorker(
```

Please also add a comment to explain what this function does.